drivers/lis2dh12/lis2dh12.c

```diff
@@ -187,6 +187,55 @@ int lis2dh12_read(const lis2dh12_t *dev, int16_t *data)
     return LIS2DH12_OK;
 }
 
+int lis2dh12_write(const lis2dh12_t *dev, uint8_t reg, uint8_t data)
+{
+    _write(dev, reg, data);
+    return LIS2DH12_OK;
+}
+
+
+/* read interrupt INT_X (INT_1 or INT_2) */
+int lis2dh12_set_interrupt(const lis2dh12_t *dev, int_params_t params, uint8_t INT_X)
+{
+    assert(dev && params.cfg && params.ths && params.duration && params.type);
```

Comment:
Would you need additional limit checking of the threshold and duration parameters or is the entire uint8_t range allowed? If there are tighter limits, please also document them.

Reply:
Is it required in all cases that threshold and duration are not equal to zero? This should also be included in the parameter documentation.

Reply:
7 bits for duration and threshold, zero is allowed. Added this.